src/components/List/List.stories.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import { Meta } from '@storybook/react'
+import { List } from '.'
+import {
+  Box,
+  Button,
+  Flex,
+  ListItem,
+  ListItemIcon,
+  ListItemSecondaryAction,
+  ListItemText,
+} from '..'
+
+import { Check } from '../Icons'
+
+export default {
+  title: 'Components/List',
+  component: List,
+} as Meta
+
+export const Default: React.FC = () => {
+  return (
+    <Flex css={{ background: '$background', justifyContent: 'center' }}>
+      <Box css={{ background: '$paper', minWidth: '500px', margin: '$3' }}>
+        <List as="nav" aria-label="main mailbox folders">
+          <ListItem interactive>
+            <ListItemIcon>
+              {/* drafts */}
+              <Check />
+            </ListItemIcon>
+            <ListItemText text="Inbox" />
+          </ListItem>
+          <ListItem interactive>
+            <ListItemIcon>
+              {/* inbox */}
+              <Check />
+            </ListItemIcon>
+            <ListItemText text="Drafts" />
+          </ListItem>
+        </List>
+        <hr />
+        <List as="nav" aria-label="secondary mailbox folders">
+          <ListItem interactive>
+            <ListItemText text="Trash" />
+          </ListItem>
+          <ListItem interactive>
+            <ListItemText text="Spam" />
+          </ListItem>
+        </List>
+      </Box>
+    </Flex>
+  )
+}
+
+export const WithIcons: React.FC = () => {
+  return (
+    <Flex css={{ background: '$background', justifyContent: 'center' }}>
+      <Box css={{ background: '$paper', minWidth: '500px', margin: '$3' }}>
+        <List>
+          <ListItem interactive>
+            <ListItemIcon>
+              {/*
```

Comment:
did you have any insight in to what was causing this?

Reply:
`<ListItem interactive>` and `<Checkbox/>` are both buttons.

Nesting buttons might be what causes the browser to hang.